### ISSUE TITLE
Memory leak in Magick::Image#sketch

### ISSUE THREAD

**Issue by Watson1978** [rmagick/rmagick#394]
When you use `Magick::Image#sketch`, the method increase memory usage continuously.
I think this is ImageMagick issue.

ImageMagick `SketchImage()` API has same two line

```c
random_view=AcquireAuthenticCacheView(random_image,exception);
```

in 
https://github.com/ImageMagick/ImageMagick6/blob/48f354c142ab3985b8113e9ea96625c5fbc1a422/magick/fx.c#L4572

and

https://github.com/ImageMagick/ImageMagick6/blob/48f354c142ab3985b8113e9ea96625c5fbc1a422/magick/fx.c#L4577

The redundant line causes memory leak in ImageMagick, not RMagick :(

* Sample code.

```
require 'rmagick'

image = Magick::Image.new(20,20)

10000.times do |i|
  image.sketch(1, 1, 1)

  GC.start if i % 100 == 0
end

GC.start
rss = `ps -o rss= -p #{Process.pid}`.to_i / 1024
puts "Process: #{Process.pid}: RSS = #{rss} MB"
```

**Comment by Watson1978** [comment]
Reported at https://github.com/ImageMagick/ImageMagick6/issues/36

**Comment by Watson1978** [comment]
I confirmed this problem fixed with ImageMagick-6.9.10-35.
